tools/get_flops.py

```diff
@@ -45,9 +45,12 @@ def main():
             f'with {model.__class__.__name__}')
 
     flops, params = get_model_complexity_info(model, input_shape)
+
     split_line = '=' * 30
     print(f'{split_line}\nInput shape: {input_shape}\n'
           f'Flops: {flops}\nParams: {params}\n{split_line}')
+    print('!!!If you are testing a recurrent network, please divide the '
```

Comment:
Why it should be divided?

Reply:
Since a recurrent network (e.g. BasicVSR) takes N frames as inputs and outputs N frames, therefore the FLOPs for each frame should be the output FLOPs divided by N.

Reply:
Okay, print this info only when the input is N frames, and change "recurrent network" to some other name. N frame to N frame does not have to be recurrent